### PR TITLE
ES: Translation for security

### PR DESCRIPTION
**Pull request by UlisesGascon**
This is my first PR to do a full page translation. I am not sure 100% for some terms translation to Spanish but I tried to be very literal and I keep some english terms that are hard to translate to proper Spanish.

Please review it @nodejs/nodejs-es 🙏

### Main changes
- Updated `locale/es/docs/guides/index.md` to match `locale/en/docs/guides/index.md` content
- Added Spanish translation for Security Guidelines

### Context

- Refs: https://github.com/nodejs/nodejs.org/pull/4896.
- [Slack discussion](https://openjs-foundation.slack.com/archives/C03Q9MS3KFB/p1668269451339869)

### CODE REVIEW

**Comment by ghost** [comment]
Ping @nodejs/nodejs-es , plz have a review.

**Review by AugustinMauroy — APPROVED** [review]
✨LGTM✨

File: locale/es/docs/guides/security/index.md
Comment (vrunoa) [inline locale/es/docs/guides/security/index.md]: **suggestion:** `Ataques explicados: ilustrar y documentar con lenguaje sencillo con algunos ejemplos código`  --> `Ataques explicados: ilustrar y documentar en lenguaje sencillo con algunos ejemplos código`.

File: locale/es/docs/guides/security/index.md
Comment (vrunoa) [inline locale/es/docs/guides/security/index.md]: **suggestion:** `los ataques que estamos mencionando en el modelo de amenazas.` --> `los ataques mencionandos en el modelo de amenazas.`

File: locale/es/docs/guides/security/index.md
Comment (vrunoa) [inline locale/es/docs/guides/security/index.md]: **suggestion:** `la aplicación es demasiado laxa` --> `la aplicación no es estricta`

**Review by vrunoa — APPROVED** [review]
Small suggestions

**Comment by ovflowd** [comment]
@AugustinMauroy I know you have the best intentions with your LGTM, but if you're not part of the Website WG or not a Spanish translator, avoid stamping LGTM's as it gives a wrong impression that this PR got approved 🙈

File: locale/es/docs/guides/security/index.md
Comment (UlisesGascon) [reply locale/es/docs/guides/security/index.md]: Fix in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)

File: locale/es/docs/guides/security/index.md
Comment (UlisesGascon) [reply locale/es/docs/guides/security/index.md]: Fixed in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)

File: locale/es/docs/guides/security/index.md
Comment (UlisesGascon) [reply locale/es/docs/guides/security/index.md]: Fixed in [c4619df](https://github.com/nodejs/nodejs.org/pull/4932/commits/c4619df7c402460da9dff8c71ce9e3205de81181)

**Comment by UlisesGascon** [comment]
Thanks for the suggestions @vrunoa! 🙌

**Review by eafelix — APPROVED** [review]

**Comment by UlisesGascon** [comment]
Can we merge it? :)

**Review by vrunoa — APPROVED** [review]

**Comment by vrunoa** [comment]
@UlisesGascon I think someone from @nodejs/website still needs to approve it ? or maybe @nodejs/translators ?

**Comment by UlisesGascon** [comment]
mmm... not sure, first time I do a translation 🤔

**Review by ovflowd — APPROVED** [review]

**Pull request merged**
